client/tests/e2e/specs/test_greet_device_modal.ts

```diff
@@ -33,7 +33,7 @@ describe('Greet a new device', () => {
     cy.window().then((win) => {
       win.navigator.clipboard.readText().then((text) => {
         // cspell:disable-next-line
-        expect(text).to.eq('parsec://parsec.example.com/MyOrg?action=claim_device&token=9ae715f49bc0468eac211e1028f15529');
+        expect(text).to.eq('parsec://parsec.example.com/Org?action=claim_device&token=9ae715f49bc0468eac211e1028f15529');
```

Comment:
How does this check work exactly ? The expected text is `parsec://example.parsec.cloud/Org?[...]` yet we check for `parsec.example.com`, and the test is valid with both `Org` and `MyOrg` 🤔

Reply:
Oh. It's even worth than I thought. I changed the `MyOrg` to `Org` because the test failed. I was very surprised to see it fail suddenly, I didn't change any of that code. But you're right, it should still fail because of the host.